### ISSUE TITLE
Any special character sequence is replaced when loading file from Pico/Pico W - VSCode

### ISSUE THREAD

**Issue by kpg141260** [paulober/pico-mpy-com#1]
> Please provide a clear and concise description of the bug along with logs

## What are the steps to reproduce this issue?
1. Create a string operation such as string.strip('\n') or bytearray a[5] = '\x0A'
2. Save the file with the instructions to the Pico
3.Close the file in VSCode
4. Open the file again
5. In the editor the code now reads: 
string.strip('\
')
or bytearray a[5] = '\
'
The newline char in the source code has actually been replaced by a real newline character

## What happens?
see above

## What were you expecting to happen?
That the load process does nor strip my source code newline or hex values and replaces them in the source code with the actual function of the character. Basically DONT change my source code!

## Any logs, error output, etc?
*(To get extension logs see Terminal > Output > Extension Host and copy output concerning pico-w-go or micropico)*
*(If it’s long, please paste to https://gist.github.com and insert the link here)*

## Any other comments?
Here is an example from my code after loading:
        if not pin_resv_low.value() and pin_resv_top.value():
            # Print the current date in the format: month/day/year hours:minutes:seconds and append a newline
            _dh.time_print (prefix = 'ERROR: ', divisor = '-', end=f'{_msg.get("msg_sensor")}\ <---- this here should be \n
', sev=sev.ERROR )  <-- this is what happens after  loading the file.

This never happened before.

## Which version of MicroPico are you using?
MicroPython v1.22.2 on 2024-02-22; Raspberry Pi Pico with RP2040

## Support info

Copy this from the `Help -> Info/About -> Copy` (`Code -> About Visual Studio Code -> Copy` on macOS) option in Visual Studio Code:

```
Version: 1.93.0 (user setup)
Commit: 4849ca9bdf9666755eb463db297b69e5385090e3
Date: 2024-09-04T13:02:38.431Z
Electron: 30.4.0
ElectronBuildId: 10073054
Chromium: 124.0.6367.243
Node.js: 20.15.1
V8: 12.4.254.20-electron.0
OS: Windows_NT x64 10.0.22631
```

**Comment by paulober** [comment]
@kpg141260 Should be fix with the next release.

**Comment by kpg141260** [comment]
Oh cool, I just wasted a lot of time correcting the entries again and again as I'm in the middle of debugging a project, so need to start, edit, save, load, correct the newlines again and so on.
But hey, no worries as long as you can fix it I'm happy and other will be too.
By the way, great job on the extension!

**Comment by kpg141260** [comment]
Hi Paul,Thanks for the quick turnaround.I’m impressed.Kind regards,Peter GosslerOn 11 Sep 2024, at 20:03, Paul ***@***.***> wrote:﻿
Closed #1 as completed via 1489cf6.
